requirements-integration.txt

```diff
@@ -1,28 +1,29 @@
 # Copyright (C) 2023 Berkhan Berkdemir <berkhan@tekdanisman.com>
+# Copyright (C) 2024 Maciej Delmanowski <drybjed@gmail.com>
 # Copyright (C) 2024 DebOps <https://debops.org/>
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 # List of Python packages used by GitHub CI pipeline for continuous integration
 
-ansible==8.0.0a1
```

Comment:
This was done because in the past we thought Ansible broke the API. See https://github.com/debops/debops/issues/2442#issuecomment-1787160157